bucket/veusz.json

```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://veusz.github.io",
+    "description": "Scientific plotting and graphing program, designed to produce publication-ready 2D and 3D plots.",
+    "license": "GPL-2.0-or-later",
+    "version": "3.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/veusz/veusz/releases/download/veusz-3.1/veusz-3.1-windows-setup.exe#/dl.7z",
```

Comment:
Where is said this file is 64bit only?

Reply:
sorry this software is works with 32bit
my computer is 64bit so i thought this works with 64bit

how do i fix it?
should i separate 64bit and 32bit using same url or not ?

Reply:
Just remove architecture property. And keep `url` and `hash` in root of manifest

Reply:
updated
dcaeb49